src/Console/GenerateSwaggerDocCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Mtrajano\LaravelSwagger\Console;
+
+use Illuminate\Console\Command;
+use Mtrajano\LaravelSwagger\FormatterManager;
+use Mtrajano\LaravelSwagger\Generator;
+use Mtrajano\LaravelSwagger\SwaggerDocsManager;
+
+class GenerateSwaggerDocCommand extends Command
+{
+    /**
+     * @var SwaggerDocsManager
+     */
+    private $swaggerDocsManager;
+
+    public function __construct(SwaggerDocsManager $swaggerDocsManager)
+    {
+        parent::__construct();
+
+        $this->swaggerDocsManager = $swaggerDocsManager;
+    }
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'laravel-swagger:generate
+                            {--format=json : The format of the output, current options are json and yaml}
+                            {--all-versions : Generate swagger docs for all versions}
+                            {--api-version= : The version of the swagger docs. Uses defaultVersion by default}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Automatically generates a swagger documentation file for this application';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     * @throws \Mtrajano\LaravelSwagger\LaravelSwaggerException
+     * @throws \ReflectionException
+     */
+    public function handle()
+    {
+        $versions = $this->getVersionsConfigToGenerate();
+
+        foreach ($versions as $versionConfig) {
+            $versionConfig['title'] = config('laravel-swagger.title');
+            $versionConfig['description'] = config('laravel-swagger.description');
+            $versionConfig['host'] = config('laravel-swagger.host');
+
+            $docs = (new Generator($versionConfig, $versionConfig['basePath']))->generate();
+
+            $format = $this->option('format');
+
+            $formattedDocs = (new FormatterManager($docs))
+                ->setFormat($format)
+                ->format();
+
+            $fileName = $this->swaggerDocsManager->generateSwaggerFileName(
+                $versionConfig['appVersion'],
+                $format
+            );
+
+            file_put_contents(public_path($fileName), $formattedDocs);
+        }
+    }
+
+    /**
+     * Get versions config to generate swagger docs.
+     *
+     * @return array
```

Comment:
When I generate the method doc block to add a description, for example, the IDE (PHPStorm) generate automatically the return/params/throws types 😄 

Personally, in Laravel packages, I like follow the framework [contribuition guide](https://laravel.com/docs/6.x/contributions#coding-style) standards.

> The reason a lot of the methods I had before weren't type hinted is because I was trying to support versions of php < 7, but those versions should be long dead

Like said in https://github.com/mtrajano/laravel-swagger/pull/24#discussion_r375185920 we changed it. But I think we can discuss the best abordage to do it.